### PR TITLE
Increase time out for jest before/after hooks

### LINKED ISSUE #1546: Figure out what is wrong with the tests
Putting I  a setTimeout of 200ms into createTables causes all sorts of catastrophes.

This should of course work. I'm figuring this is the same issue as the pipeline due to it being timing in the environment.

### PR DESCRIPTION
**Pull request by diversemix**
#### Background

Prevent the tests from timing out in the pipeline.

#### Any relevant tickets

Closes #1546

#### How has this been tested?

This would be a bit "meta" :stuck_out_tongue_closed_eyes:

### CODE REVIEW

**Comment by diversemix** [comment]
Third time of running - if it works then I think its good to go

File: server/jest-setup.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ const replaySetup = require('../test/helpers/replay-setup')
 
 replaySetup('success')
 
+// The pipeline complains at the default of 5s - so double it!
```
Comment (Cooryd) [inline server/jest-setup.js:6]: Might be useful to comment what is meant by default. or something like `//This increases the interval between the before and after hooks and tests from 5s to 10s.`

Reviewers see:
https://jestjs.io/docs/en/jest-object#jestsettimeouttimeout

Comment (diversemix) [reply server/jest-setup.js:6]: Ooops - I did plan on putting that link in the comment - let me do that... 

What do you mean by:
> Might be useful to comment what is meant by default

**Review by Cooryd — APPROVED** [review]
Solution looks good and makes sense. Is this scalable over time? or will we likely need to increase CI resource / increase this `setTimeout` value as the product and test suites expand?

File: server/jest-setup.js (hunk 1 above)
Comment (Cooryd) [reply server/jest-setup.js:6]: As in what the time relates to. I suppose the docs link would be sufficient explanation

**Comment by diversemix** [comment]
My best guess is that its a function of the infrastructure and the number of tests in a given file - given that `afterAll` does a `db.destroy()`

File: server/jest-setup.js (hunk 1 above)
Comment (diversemix) [reply server/jest-setup.js:6]: Added the link ... the other I think is a matter of style so will leave the rest of the comment intact

**Review by hem-brahmbhatt — APPROVED** [review]

**Pull request merged**